use std::error::Error;

pub fn process(_input: &str) {
    todo!("part g");
}
